diff --git a/seeds/seed.js b/seeds/seed.js
--- a/seeds/seed.js
+++ b/seeds/seed.js
@@ -85,8 +85,8 @@ exports.seed = function (knex) {
       return knex('server_settings').insert([
         {
           stripe_api_key: 'sk_test_123',
           sendgrid_credentials: 'sg_test_456',
-          default_fee_rate: 0.5,
-          setup_complete: false
+          default_fee_rate: 0.5,
+          setup_complete: true
         }
       ])
     })
