diff --git a/src/pages/Buttons/index.tsx b/src/pages/Buttons/index.tsx
--- a/src/pages/Buttons/index.tsx
+++ b/src/pages/Buttons/index.tsx
@@ -262,1 +262,1 @@
-    description: button.description ?? 'No description',
+    description: button.description || `Payment using paymentId: ${formatId(button.paymentId || '')}`,
@@ -244,2 +244,2 @@
-                            {button.description}
+                            {button.description || `Payment using paymentId: ${formatId(button.payment_id || '')}`}

